index.html

```diff
@@ -0,0 +1,222 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <section class="wrapper">
+        <nav class="toolbar">
+            <ul>
+                <li class="logo"><a href="#"><b>Basir Elham</b></a></li>
+                <li class="collapse-button"><a href="#"><span class="fa fa-list"></span></a></li>
+                <li class="menu">
+                    <span>
+                        <ul>
+                            <li><a href="#"><b>Portfolio</b></a></li>
+                            <li><a href="#"><b>About</b></a></li>
+                            <li><a href="#"><b>Contact</b></a></li>
+                        </ul>
+                    </span>
+                </li>
+            </ul>
+        </nav>
+        <div class="headline">
+            <div class="content">
+                <h1>Hey There <br> I'm Basir Elham</h1>
+                <p>
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Rem saepe incidunt ut tempore ipsam cumque
+                    eveniet voluptate, eligendi cupiditate recusandae facere nam tempora quaerat maiores voluptatum
+                    quisquam minus atque unde.
+                </p>
+                <b>LET'S CONNECT</b>
+                <div class="social-media">
+                    <ul>
+                        <li><a href=""><span class="fa fa-twitter"></span></a></li>
+                        <li><a href=""><span class="fa fa-linkedin-square"></span></a></li>
+                        <li><a href=""><span class="fa fa-github-square"></span></a></li>
+                        <li><a href=""><span class="fa fa-facebook"></span></a></li>
+                    </ul>
+                </div>
+            </div>
+        </div>
```

Comment:
Some background images are missing in this part, kindly add them as we have them in the Figma design. Also, kindly add the border-radius at the bottom-left of the container just like we have it in the design. 😉 😌  

| Your Design | Figma Design |
| :---:   | :---: | 
| <img width="373" alt="image" src="https://user-images.githubusercontent.com/73854999/219464342-22bce005-a28a-4f9e-8421-7cfff95dad77.png">| <img width="373" alt="image" src="https://user-images.githubusercontent.com/73854999/219464480-a2ec186e-703f-4fd8-b199-127e555ac948.png">|